Makefile

```diff
@@ -44,8 +44,7 @@ clean:
 
 test:
 	@echo "run test"
-	@go test -race -coverprofile=coverage.txt -covermode=atomic -v ./...
-	@go tool cover -html="coverage.txt" -o "coverage.html"
+	@go test -race -v -count=1 ./task
```

Comment:
what does this count flag mean?

Reply:
It's a trick to let `go test` ignore cache.